scala-akka/README.md

```diff
@@ -0,0 +1,7 @@
+## Scala Akka
+
+This module contains articles about the Akka library in Scala.
+
+### Relevant Articles:
+
+- [Typed Akka: The Actor Model Done Right](https://www.baeldung.com/scala/todo)
```

Comment:
Let's remove this line from the readme. The team will take care of this when the article is published at some point.

Reply:
Hi @JonCook . I made the change you requested :)